### PR TITLE
Added barter npcs to sell/exchange refine ore

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: -

* **Server Mode**: renewal

* **Description of Pull Request**:

### CODE REVIEW

**Comment by Pokye** [comment]
Need change AegisName to (LI_HD_Oridecon, LI_HD_Elunium, LI_HD_Bradium, LI_HD_Carnium) in **db/re/item_db_etc.yml** ids 6910, 6911, 6327, 6906

**Comment by il3ol2ed** [comment]
Why the price is -1

**Comment by Atemo** [comment]
@il3ol2ed your screenshot is not shown

**Comment by Pokye** [comment]
@Atemo is it really free?
image

**Comment by Atemo** [comment]
Prices should be 200 and 1000 (the price of items)

**Comment by il3ol2ed** [comment]
Sorry, I don't know if it doesn't show pictures either, but thanks @Pokye for reporting this issue.
screenrAthena003

**Comment by il3ol2ed** [comment]
Thanks 🙏

File: npc/re/merchants/refine.txt [hunk 1]
```diff
@@ -582,3 +583,113 @@ malangdo,224,172,6	script	Clink#mal_normal	544,{
...
+-	marketshop	market_refine_payon	-1,1010:-1:999999,1011:-1:999999
+-	marketshop	market_refine_morocc_in	-1,1010:-1:999999,1011:-1:999999
+-	marketshop	market_refine_alberta_in	-1,1010:-1:999999,1011:-1:999999
+-	marketshop	market_refine_yuno_in01	-1,1010:-1:999999,1011:-1:999999
+-	marketshop	market_refine_ein_in01	-1,1010:-1:999999,1011:-1:999999
+-	marketshop	market_refine_lhz_in02	-1,1010:-1:999999,1011:-1:999999
+
+prt_in,63,69,3	script	Dietrich#ns_prt	4_M_02,{
+	mes "[" + strnpcinfo(1) + "]";
+	mes "We sell and exchange various smelting ores used in smelting.";
```
Comment (aleos89) [inline npc/re/merchants/refine.txt:647]:
```suggestion
	mes "We sell and exchange various ores used in smelting.";
```

**Review by aleos89 — APPROVED** [review]

**Pull request merged**